src/org/mozilla/javascript/NativeString.java

```diff
@@ -941,6 +946,47 @@ protected int findPrototypeId(Symbol k)
         return 0;
     }
 
+    /**
+     * <h1>String.raw (template, ...substitutions)</h1>
+     * <p>22.1.2.4 String.raw [Draft ECMA-262 / April 28, 2021]</p>
+     */
+    private static CharSequence js_raw(Context cx, Scriptable scope, Object[] args) {
+        final Object undefined = Undefined.instance;
+        /* step 1-2 */
+        Object arg0 = args.length > 0 ? args[0] : undefined;
```

Comment:
I'm not sure if the "undefined" case is being tested here.

Reply:
`template-not-object-throws.js` tests `String.raw(undefined)`. Is that good enough or do you want a `String.raw()`, too?